src/parser/warrior/fury/modules/buffdebuff/Enrage.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import Analyzer from 'parser/core/Analyzer';
+
+import AbilityTracker from 'parser/shared/modules/AbilityTracker';
+import StatTracker from 'parser/shared/modules/StatTracker';
+import Events from 'parser/core/Events';
+import { SELECTED_PLAYER } from 'parser/core/EventFilter';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage, formatNumber, formatThousands } from 'common/format';
+import StatisticBox from 'interface/others/StatisticBox';
+import calculateEffectiveDamage from 'parser/core/calculateEffectiveDamage';
+
+class Enrage extends Analyzer {
+  static dependencies = {
+    statTracker: StatTracker,
+    abilityTracker: AbilityTracker,
+  }
+
+  totalDamage = 0;
+  damage = 0;
+
+  constructor(...args) {
+    super(...args);
+
+    this.addEventListener(Events.damage.by(SELECTED_PLAYER), this.onPlayerDamage);
+  }
+
+  onPlayerDamage(event) {
+    if (this.selectedCombatant.hasBuff(SPELLS.ENRAGE.id)) {
+      this.damage += calculateEffectiveDamage(event, this.statTracker.currentMasteryPercentage);
+      this.totalDamage += event.amount;
+    }
+  }
+
+  get uptime() {
+    return this.selectedCombatant.getBuffUptime(SPELLS.ENRAGE.id) / this.owner.fightDuration;
+  }
+
+  get damageTotal () {
```

Comment:
This is a little nit picky, but you are calling this damageTotal but you arent actually outputing a total amount of damage, you are outputting the Damage Per Second increase. Might want to rename this to be more descriptive of what you are actually getting here.